IdeasTracker/IdeasTracker/IdeasTrackerApp.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  IdeasTrackerApp.swift
+//  IdeasTracker
+//
+//  Created by Wen Yu Ge on 2024-04-19.
+//
```

Comment:
Do we need these style comments? is it added by xcode

Reply:
Ah yeah xcode we should remove